src/lib/util/Games/TriviaManager.ts

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
Please unit test this (so coveralls doesn't complain). You can use `nock` for this (and I think it's already installed).